include/swift/SIL/SILConstants.h

```diff
@@ -556,7 +556,7 @@ class SymbolicValue {
   void dump() const;
 };
 
-static_assert(sizeof(SymbolicValue) == 2 * sizeof(void *),
+static_assert(sizeof(SymbolicValue) == 2 * sizeof(uint64_t),
               "SymbolicValue should stay small");
 static_assert(std::is_pod<SymbolicValue>::value,
```

Comment:
@marcrasi While here, I suggest changing this to `is_trivially_copyable` or `is_trivial` or `is_standard_layout`, whichever is actually being relied upon. See https://en.cppreference.com/w/cpp/named_req/PODType for more details.

Reply:
I made a PR: https://github.com/apple/swift/pull/27521